tests/TestCase.php

```diff
@@ -33,6 +39,11 @@ protected function setUp(): void
         Auth::setUser(User::query()->where('name', 'Administrator')->first());
 
         $this->travelTo(Carbon::now());
+
+        // Keep time ticking across database calls
+        DB::listen(function () {
+            $this->travel(1)->second();
+        });
```

Comment:
does this effectively replace the need to call `travel()` anywhere else in the test suite since it happens every time we make a DB call? if so might be a light refactor we can do to clean up tests later. very low priority

Reply:
Yah, we could probably remove other references to travel with this.